ci/tasks/deploy.sh

```diff
@@ -7,4 +7,7 @@ cf auth
 
 cf t -o $CF_ORG -s $CF_SPACE
 
-cf push $CF_APP_NAME -f $CF_MANIFEST --strategy rolling --vars-file $CF_VARS_FILE
\ No newline at end of file
+cf push $CF_APP_NAME -f $CF_MANIFEST \
+  --strategy rolling \
+  --vars-file $CF_VARS_FILE \
+  --stack cflinuxfs4
```

Comment:
Should we make `cflinuxfs4` a parameter and put it in as an env var in the CI pipeline? Or is that unnecessary overhead?

Reply:
If the system can be expected to work across multiple stacks and which you use is a deployment/operational decision then I think it would be fine to parameterize via the environment.

If the system is tightly associated with `cflinuxfs4` and uncertain to test and work reliably across arbitrary stacks I think it makes more sense to leave that setting as committed code.

(I'm not sure which of these situations holds, @apburnes.)

Reply:
Add CF_STACK to the pipeline.yml and updated the deploy script to use `$CF_STACK`